src/components/collection/summary/SummaryDetails.tsx

```diff
@@ -126,10 +127,11 @@ const SummaryDetails: React.FC<SummaryDetailsProps> = ({
     ?.filter((c) => c.category.type === CategoryType.Discipline)
     ?.map((c) => c.category);
 
-  // TODO: should use item language
-  const languages = itemCategories
-    ?.filter((c) => c.category.type === CategoryType.Language)
-    ?.map((c) => c.category);
+  let langValue = langs[DEFAULT_LANG];
+  if (collection.lang in langs) {
+    // @ts-ignore
+    langValue = langs[collection.lang];
+  }
```

Comment:
Why do you need `@ts-ignore`? Also what do you think about that?

```suggestion
  const lang = collection.lang in langs ? collection.lang : DEFAULT_LANG;
  const langValue = langs[lang];
```